testsuite/features/build_validation/init_clients/slemicro52_minion.feature

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# Beware: After altering the system e.g. package installation/removal, the system
+#         has to be rebooted to take effect due to the nature of SLE Micro.
+
+@slemicro52_minion
+Feature: Bootstrap a SLE Micro 5.2 Salt minion
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Bootstrap a SLE Micro 5.2 minion
+    When I follow the left menu "Systems > Bootstrapping"
+    Then I should see a "Bootstrap Minions" text
+    When I enter the hostname of "slemicro52_minion" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "linux" as "password"
+    And I select "1-slemicro52_minion_key" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies" if present
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+    And I wait until onboarding is completed for "slemicro52_minion"
+
+  Scenario: Reboot the SLE Micro 5.2 minion and wait until reboot is completed
+    When I reboot the "slemicro52_minion" minion through SSH
+
+  Scenario: Check the new bootstrapped SLE Micro 5.2 minion in System Overview page
+    When I follow the left menu "Salt > Keys"
+    Then I should see a "accepted" text
+    And the Salt master can reach "slemicro52_minion"
+
+  # 2 reboots are necessary after bootstrapping
```

Comment:
Do you warn about two reboots but I can only see one in the code?

Reply:
We have the first reboot directly after bootstrapping and then this one. In case anyone wonders why there are 2 reboots nearly next to each other.

Reply:
AH. Got it.